photon-api/src/integTest/scala/com/linkedin/photon/ml/function/DistributedObjectiveFunctionIntegTest.scala

```diff
@@ -66,38 +66,37 @@ class DistributedObjectiveFunctionIntegTest extends SparkTestUtils {
    * @return Anonymous functions to generate the loss function and training data for the gradient tests
    */
   @DataProvider(parallel = true)
-  def getDifferentiableFunctions: Array[Array[Object]] = diffTasks
-    .flatMap {
-      case TaskType.LOGISTIC_REGRESSION =>
-        treeAggregateDepths.flatMap { treeAggDepth =>
-          def lossFuncBuilder = (sc: SparkContext) =>
-            DistributedGLMLossFunction(sc, NO_REG_CONFIGURATION_MOCK, treeAggDepth)(LogisticLossFunction)
-          def lossFuncWithL2Builder = (sc: SparkContext) =>
-            DistributedGLMLossFunction(sc, L2_REG_CONFIGURATION_MOCK, treeAggDepth)(LogisticLossFunction)
+  def getDifferentiableFunctions: Array[Array[Object]] = diffTasks.flatMap {
+    case TaskType.LOGISTIC_REGRESSION =>
+      treeAggregateDepths.flatMap { treeAggDepth =>
+        def lossFuncBuilder =
+          () =>DistributedGLMLossFunction( NO_REG_CONFIGURATION_MOCK, treeAggDepth)(LogisticLossFunction)
```

Comment:
I think need an additional space after `(` and need another space after `=>`. Same for line 75, 85, 87, 97, 99, etc.

Reply:
Will do - looks like something went wrong with my style